conftest.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+from unittest import mock
+import app
+import boto3
+from dcicutils.env_utils import EnvUtils
+from dcicutils.env_manager import EnvManager
+from dcicutils.misc_utils import override_environ, ignored, PRINT
+from foursight_core.app_utils import AppUtils
+from foursight_core.sqs_utils import SQS
+from foursight_core.environment import Environment
+from foursight_core.decorators import Decorators
+
+
+GLOBAL_ENV_BUCKET = 'GLOBAL_ENV_BUCKET'
+FOURSIGHT_PREFIX = 'foursight-core-simulated'
+SIMULATED_GLOBAL_ENV_BUCKET = FOURSIGHT_PREFIX + '-envs'
+DEV_ENV = 'simulated'
+ES_HOST = 'https://search-fourfront-testing-opensearch-kqm7pliix4wgiu4druk2indorq.us-east-1.es.amazonaws.com'
+SIMULATED_ENV_CONFIG = {
+    'simulated': {
+        'fourfront': 'dummy-url',
+        'es': ES_HOST,  # maybe dont even need this?
+        'ff_env': 'fourfront-simulated',
+        'bucket': 'foursight-core-simulated-results',
+    },
+    'main.ecosystem': {
+        'public_url_table': [
+            {
+                'name': 'simulated',
+                'url': 'dummy-url',
+                'host': 'dummy-host',
+                'environment': 'fourfront-simulated'
+            }
+        ],
+        'foursight_bucket_table': {
+            'simulated': {
+                'dev': 'foursight-core-simulated-results'
+            }
+        }
+    }
+}
+SIMULATED_ENV_HEALTH_PAGE = {
```

Comment:
I wonder if we could somehow unify this stuff with the sample scenarios already available on EnvUtils.